apps/frontend/package.json

```diff
@@ -2,7 +2,8 @@
 	"name": "@skyra/nayre-web",
 	"scripts": {
 		"build": "nuxt build",
-		"dev": "nuxt dev -p 3200",
+		"dev": "yarn build && yarn start",
+		"watch": "nuxt dev -p 3200",
```

Comment:
Mind this @kyranet. In order to ensure that scripts are identical across package.jsons `dev` is now `watch` for the frontend.